resources/gcp/compute/subnetwork.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: compute.gcp.crossplane.io/v1alpha3
+kind: Subnetwork
+metadata:
+  name: subnetwork
+spec:
+  name: subnetwork
+  region: $(REGION)
```

Comment:
What testing has been done for this style of variable substitution?

Reply:
I manually tested this. In the `kustomizeconfig.yaml` the field should be declared as candidate for variable substition, otherwise it won't be touched. Then in the overlay you define an array of `Var` objects that refer to the custom resource field. You can find how this works here https://github.com/kubernetes-sigs/kustomize/blob/master/examples/transformerconfigs/crd/README.md

Reply:
Awesome